packages/app/public/static/locales/zh_CN/admin.json

```
@@ -238,6 +239,7 @@
 		}
   },
   "notification_settings": {
+    "notification_settings": "Notification Settings",
```

```suggestion
    "notification_settings": "通知设置",
```
ですかね 👀 

zh_CH/translatoin.json の方は削除お願いします!